spec/registry/unicode.md

```diff
@@ -0,0 +1,63 @@
+# MessageFormat 2.0 Unicode Registry
+
+The `u:` namespace is reserved for use by the Unicode Consortium.
+
+## Options
+
+This section describes common options which each implementation SHOULD support
+for all _functions_ and _markup_.
+
+### `u:id`
+
+A string value that is included as an `id` or other suitable value
+in the formatted parts for the _placeholder_,
+or any other structured formatted results.
+
+Ignored when formatting a message to a string.
+
+Accepts string values, or values which can be stringified without error.
+For other values, a _Bad Option_ error is emitted
+and the `u:id` option is ignored.
+
+### `u:locale`
+
+A comma-delimited list of BCP 47 language tags,
+or an implementation-defined list of such tags.
+
+Replaces the _locale_ defined in the _function context_ for this _expression_.
+The value is ignored when set on _markup_.
+
+During processing, the `u:locale` option
+is always removed from the resolved mapping of _options_.
+
+Values matching the following ABNF are always accepted:
+```abnf
+u-locale-option = langtag *([s] "," [s] langtag)
+```
+using `langtag` as defined in [RFC 5646](https://www.rfc-editor.org/rfc/rfc5646).
```

Comment:
So let's not link to a specific RFC! BCP47 only has the one `langtag` production in it. Link to BCP47.

Reply:
```suggestion
using `langtag` as defined in [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt).
Note that `langtag` is the rule for "normal language tags",
and does not include private-use or grandfathered tags.
```

Reply:
https://www.rfc-editor.org/rfc/bcp/bcp47.txt should be https://www.rfc-editor.org/bcp/bcp47